SWTableViewCell/PodFiles/SWTableViewCell.m

```diff
@@ -658,7 +699,15 @@ - (void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoi
         }
         else
         {
-            _cellState = kCellStateRight;
+            if (velocity.x > kLongSwipeVelocityThreshold)
+            {
```

Comment:
changed all to braces on new line. hard to see becvause the file in and of itself is inconsistent.. : )